tests/identityManager.test.ts

```
@@ -0,0 +1,52 @@
+import * as chai from 'chai'
+import * as sinonChai from 'sinon-chai'
+import * as testKeys from './data/keys'
+import { IdentityManager } from '../ts/identityManager/identityManager'
+chai.use(sinonChai)
+const expect = chai.expect
+
+describe('IdentityManager', () => {
+  const identityManager = IdentityManager.create(testKeys.default.testSeed)
+
+  describe('static create', () => {
+    it('should create an instance of IdentityManager with correct default keys schema', () => {
+      const keySchema = identityManager.getSchema()
+
+      expect(keySchema).to.have.property('jolocomIdentityKey')
```

Can these be omitted, and simply check for the value, e.g. if `jolocomIdentityKey` is not in the schema then

```javascript
expect(keySchema.jolocomIdentityKey).to.equal('m/73\'/0\'/0\'/0')
```
should already fail, given that the result will be `undefined`

---

Also the explicit path could perhaps be replaced with 
```javascript
keyTypes.jolocomKey
```
from the enum defined in `ts/index.ts`